hex/spec/fixtures/projects/private_repo/README.md

```diff
@@ -0,0 +1,26 @@
+# Dependabot
```

Comment:
Let's not include all the files from this git repo in our repository. We can build it from source instead maybe? We might consider forking that codebase and building from the forked source.